alarm_table_defs.hpp

```diff
@@ -39,6 +39,22 @@
 
 #include "alarmdefinition.h"
 
+// Unique key for alarm table definitions is comprised of alarm index and
```

Comment:
Why have you moved this code up? (It makes it harder to review, as I can't tell if it's just been moved, or if you've moved it and changed the functionality too.)